src/libraries/tests.proj

```diff
@@ -354,6 +354,9 @@
   <ItemGroup Condition="'$(RunSmokeTestsOnly)' == 'true'">
     <ProjectReference Include="$(MSBuildThisFileDirectory)System.Runtime\tests\System.Runtime.Tests.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(RunSmokeTestsOnly)' == 'true' and '$(TargetOS)' == 'Android'">
```

Comment:
This is too low-value test for the smoke test subset. Is this just a temporary change?

Reply:
Yes, its intended to be temporary and removed before merge

Reply:
it's going to go to System.Net.Security UnitTests project.